File: HoWo5_6/ShopService/PhoneMag.java
package HoWo5_6.ShopService;

import java.util.ArrayList;

import HoWo5_6.Phone.Phone;

public class PhoneMag  {

    private ArrayList<Phone> priceList;
    private double money;


    public PhoneMag(double money) {
        this.money = money;
        this.priceList = new ArrayList<Phone>();
    }

    public PhoneMag(ArrayList<Phone> priceList) {
        this.priceList = priceList;
        this.money = 0.0;
    }

    public PhoneMag(ArrayList<Phone> priceList, double money) {
        this.priceList = priceList;
        this.money = money;
    }

    public ArrayList<Phone> getPriceList() {
        return this.priceList;
    }

    public void setPriceList(ArrayList<Phone> priceList) {
        this.priceList = priceList;
    }

    public double getMoney() {
        return money;
    }

    public void setMoney(double money) {
        this.money = money;
    }
}

File: HoWo7/ShopService/PhoneMag.py
from Phone import Phone


class PhoneMag:
    priceList: list[Phone]
    money: int  # double

    def __init__(self, priceList=[], money=0.0):
        self.money = money
        self.priceList = priceList

    def getPriceList(self) -> list[Phone]:
        return self.priceList

    def setPriceList(self, priceList: list[Phone]):
        self.priceList = priceList

    def getMoney(self):  # double
        return self.money

    def setMoney(self, money):
        self.money = money